lib/matplotlib/textpath.py

```diff
@@ -358,11 +349,21 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     warnings.warn("No supported encoding in font (%s)." %
                                   font_bunch.filename)
 
+                # Character is a glyph and needs to be mapped to corresponding index
                 if charmap_name == "ADOBE_STANDARD" and font_bunch.encoding:
+                    use_glyph = True
                     enc0 = dviread.Encoding(font_bunch.encoding)
-                    enc = {i: self._adobe_standard_encoding.get(c, None)
-                           for i, c in enumerate(enc0.encoding)}
+
+                    # Make a list of each glyph by splitting the encoding
+                    enc0_list = []
```

Comment:
this is the same as `enc0_list = [e.split('/') for e in enc0.encoding]` ?

Why do this splitting?